apps/custom_actions/models.py

```diff
@@ -70,7 +74,18 @@ def get_operations_by_id(self):
         return {op.operation_id: op for op in self.operations}
 
 
-class CustomActionOperation(models.Model):
+class CustomActionOperationManager(VersionsObjectManagerMixin, models.Manager):
+    pass
+
+
+class CustomActionOperation(models.Model, VersionsMixin, VersioningMixin):
```

Comment:
Maybe `VersioningMixin` should be collapsed onto `BaseModel`? There's only 1 method in there

Reply:
That wouldn't help here since this model doesn't extend `BaseModel` but maybe it should?

Reply:
I don't see why not? My philosohy is to rather have those created_at / updated_at timestamps from BaseModel than not.